<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protocol Emulator V0.03</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tab_ExchangeList">
       <attribute name="title">
        <string>Exchange List</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>501</width>
          <height>391</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>12</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>AI Change Request(0x01)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>AI All Change Request(0x02)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Accumulator Reset(0x03)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Accumulator Stored Value Request(0x04)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Overall Check Request(0x13)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Remote Control(0x19)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Analog Set-Point Setting(0x1A)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Digital Output Setting(0x1B)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Time Bias Computation Request(0x1C)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Clock Synchronization(0x1D)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>RTU Identification(0x20)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Remote Initialization(0x21)</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Check</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Flags</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Freq. (Secs.)</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>440</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Run Once</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_PointSummary">
       <attribute name="title">
        <string>Point Summary</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_LineMonitor">
       <attribute name="title">
        <string>Line Monitor</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>640</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>640</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Resume</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Message">
       <attribute name="title">
        <string>Message</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_EventLog">
       <attribute name="title">
        <string>Event Log</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_TrafficIO">
       <attribute name="title">
        <string>Traffic I/O</string>
       </attribute>
       <widget class="QTableWidget" name="lineMonitorTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>911</width>
          <height>661</height>
         </rect>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>Raw Packege</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Protocol Parsing</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>  Pause</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionRun_Once"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_2"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionTask_Mode"/>
    <addaction name="actionExchange_Mode"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPoint_List"/>
    <addaction name="actionExchange_List"/>
    <addaction name="actionLine_Monitor"/>
    <addaction name="separator"/>
    <addaction name="actionLine_Analyzer"/>
    <addaction name="separator"/>
    <addaction name="actionEvent_Log"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/Refresh.png</normaloff>:/images/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run continuiously</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/Stop playing.ico</normaloff>:/images/Stop playing.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Running</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string extracomment="thytrytryr">&amp;Properties</string>
   </property>
   <property name="toolTip">
    <string extracomment="rwqrwqerq">Configure communication settingsserial port</string>
   </property>
   <property name="statusTip">
    <string extracomment="ewqrqr"/>
   </property>
   <property name="whatsThis">
    <string extracomment="2233"/>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear console message</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTask_Mode">
   <property name="text">
    <string>Task Mode</string>
   </property>
  </action>
  <action name="actionExchange_Mode">
   <property name="text">
    <string>Exchange Mode</string>
   </property>
  </action>
  <action name="actionSelect_Protocol">
   <property name="text">
    <string>Select Protocol...</string>
   </property>
  </action>
  <action name="actionPoint_List">
   <property name="text">
    <string>Point List</string>
   </property>
  </action>
  <action name="actionExchange_List">
   <property name="text">
    <string>Exchange List</string>
   </property>
  </action>
  <action name="actionLine_Monitor">
   <property name="text">
    <string>Line Monitor</string>
   </property>
  </action>
  <action name="actionLine_Analyzer">
   <property name="text">
    <string>Line Analyzer</string>
   </property>
  </action>
  <action name="actionEvent_Log">
   <property name="text">
    <string>Event Log</string>
   </property>
  </action>
  <action name="actionRun_Once">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/Redo.png</normaloff>:/images/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Run Once</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Program</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
